### PR TITLE
[Training] Convert module to modern module structure 

### PR DESCRIPTION
**Pull request by kongtiaowang**
This pull request changes the Module class for training.

### CODE REVIEW

File: modules/training/help/training.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<h2>Training</h2>
```
Comment (driusan) [inline modules/training/help/training.md]: Headings should use the markdown style '# ' prefix, not HTML, and paragraphs need an empty line between them in markdown.

File: modules/training/php/training.class.inc [hunk 2]
```diff
@@ -0,0 +1,229 @@
+<?php
```
Comment (driusan) [inline modules/training/php/training.class.inc]: shouldn't the old file be removed too?

File: modules/training/help/training.md (hunk 1 above)
Comment (kongtiaowang) [reply modules/training/help/training.md]: @driusan 
It has been fixed.

File: modules/training/php/training.class.inc (hunk 2 above)
Comment (kongtiaowang) [reply modules/training/php/training.class.inc]: @driusan 
You are right, the old file should be removed. It is done.

File: modules/training/php/module.class.inc [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * This serves as a hint to LORIS that this module is a real module.
+ * It does nothing but implement the module class in the module's namespace.
+ *
+ * PHP Version 5
+ *
+ * @category Behavioural
+ * @package  Main
+ * @author   Xavier Lecours Boucher <xavier.lecoursboucher@mcgill.ca>
```
Comment (xlecours) [inline modules/training/php/module.class.inc]: @kongtiaowang you can put your name here

File: modules/training/php/module.class.inc [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+ * @link     https://www.github.com/aces/Loris-Trunk/
+ */
+namespace LORIS\training;
+
+/**
+ * Class module implements the basic LORIS module functionality
+ *
+ * @category Behavioural
+ * @package  Main
+ * @author   Xavier Lecours Boucher <xavier.lecoursboucher@mcgill.ca>
```
Comment (xlecours) [inline modules/training/php/module.class.inc]: here as well

**Comment by kongtiaowang** [comment]
@xlecours Please review again.

**Comment by xlecours** [comment]
@ridz1208 Can you test this? I only have a partly working training on my VM. I mean, it is evenly not working before and after this PR so it look fine but...

**Comment by ridz1208** [comment]
@xlecours @driusan Our training set-up has diverged due to additional features such as multi french support... the natvie loris module does not work for me either (even without this PR) but I am communicating with @taracampbell to see if she can give it a run.

**Comment by driusan** [comment]
@ridz1208 Why does this module exist in LORIS at all if no one is using it?

**Comment by ridz1208** [comment]
@driusan well... it does what it says, it has no project specific code and we advertise for it... so why remove it ??

CCNA has an override because we needed to add functionality on very short deadlines since this module is heavily used at the beginning of the study. We do have plans to push these improvements to the loris version including multi-lingual support !!

**Pull request merged**